roles/credential/defaults/main.yml

```
@@ -0,0 +1,8 @@
+---
+eda_credentials: []
+
+eda_configuration_credential_secure_logging: "{{ eda_configuration_secure_logging | default(false) }}"
```

```suggestion
eda_configuration_credential_secure_logging: "{{ eda_configuration_secure_logging | default(true) }}"
```

A convention we have followed in the other `infra.` repositories is that higher risk roles we apply a default of `true` for secure logging to minimise risk of credential exposure. Could you make this change please

---

Agreed and resolved.